Jellyfin.Server/Jellyfin.Server.csproj

```diff
@@ -41,6 +44,10 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.7.82" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.1.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
AFAIK this should be in the `Jellyfin.Server.Implementations` project since it is the project containing the `DbContext` and migrations